pytest_fixtures/component/taxonomy.py

```diff
@@ -88,14 +88,6 @@ def module_org_with_manifest(module_org, module_target_sat):
     return module_org
 
 
-@pytest.fixture(scope='module')
-def module_entitlement_manifest_org(module_org, module_entitlement_manifest, module_target_sat):
```

Comment:
this seems to be used in https://github.com/SatelliteQE/robottelo/blob/efabac69201ce9ddbf504ad50941ab4ed2e71341/tests/foreman/ui/test_contentview_old.py#L234

Reply:
These tests are being removed shortly as part of component eval, no reason to take this out as part of this.